spec/dummy/db/schema.rb

```diff
@@ -12,7 +12,7 @@
 
 ActiveRecord::Schema[8.0].define(version: 2024_11_14_165947) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
+  enable_extension "pg_catalog.plpgsql"
```

Comment:
Wonder why schema changed, considering there is no migrations present 🤔

Reply:
Agree, let's revert this schema change